client/reports.go

```diff
@@ -349,6 +376,7 @@ func (c *Client) threadedSyncWithServer(latestReading uint32) bool {
 			// that we can have relevant bits of logic
 			// after the loop which assume that a
 			// connection was made successfully.
+			c.EventLog.Printf("server sync gave up after 5 attempts")
```

Comment:
now that I'm reading this, it'd probably be nice to list the most recent failed sync operation in the summary as well

Reply:
There are a few things like this we can keep track of.

Reply:
I'm adding an "event stats" that allow keeping track of number of times called, and last called for an event type. This will allow us to keep track of interesting events without spamming the logs, and allow the logs to be used for errors or unusual events